src/CoamClient.js

```diff
@@ -405,8 +405,9 @@ class CoamClient {
         return this.__exec(data);
     }
 
-    getUserPermissionsForResourceType(principal, resourceType) {
-        let url = this.__buildUrl(`/auth/access-management/v1/principals/{{principal}}/permissions/{{resourceType}}`, {
+    getUserPermissionsForResourceType(principal, resourceType, include = true, permissionFilters = null) {
```

Comment:
One more, `permissionFilters` is plural. The API expects that this will be a comma separated list. However, in our library, we likely want to pass in an array.

What about we just to `permissionFilters.join(',')`?

Reply:
I think yes that is more convenient.